lib/syscall_shim/syscall_entertab.lds.S

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2024, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+
+#include <uk/arch/limits.h> /* for __PAGE_SIZE */
+#include <uk/plat/common/common.lds.h> /* for UK_SEGMENT_DATA */
+
+SECTIONS
+{
+	. = ALIGN(__PAGE_SIZE);
+
+	.uk_syscall_entertab :
+	{
+		PROVIDE(uk_syscall_entertab_start = .);
+		KEEP(*(SORT_BY_NAME(.uk_syscall_entertab[0-9])))
+		PROVIDE(uk_syscall_entertab_end = .);
+	} UK_SEGMENT_DATA
```

Comment:
Interestingly, except from `libukreloc`, we don't annotate the rest of the library-defined sections with `SEGMENT_DATA`. Same question for `exittab`.

Reply:
Yes, right, this one does not need that. .ukreloc for some reason needs that though. I had to fight some linker script shenanigans related to section order and their mapping to segments.... .ukreloc and the placement of the other relocation sections is not really conventional.

(That being said, I copy pasted this from there initially :laughing:)